pkg/agent/netpol/netpol.go

```diff
@@ -8,20 +8,42 @@ package netpol
 
 import (
 	"context"
+	"net"
 	"strings"
 	"sync"
 
 	"github.com/cloudnativelabs/kube-router/pkg/controllers/netpol"
 	"github.com/cloudnativelabs/kube-router/pkg/healthcheck"
 	"github.com/cloudnativelabs/kube-router/pkg/options"
 	"github.com/cloudnativelabs/kube-router/pkg/utils"
+	"github.com/coreos/go-iptables/iptables"
 	"github.com/k3s-io/k3s/pkg/daemons/config"
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	v1core "k8s.io/api/core/v1"
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// JoinNets joins a slice of IPNet to a comma-separated string.
```

Comment:
I believe we already have this at https://github.com/k3s-io/k3s/blob/c2e846dc16bfbcf949c245a82da2035e4f35fe18/pkg/util/net.go#L23-L24

Reply:
Fixed, thanks!